patterns/creational/factory.py

```diff
@@ -26,8 +26,7 @@
 
 
 class Localizer(Protocol):
-    def localize(self, msg: str) -> str:
-        pass
+    def localize(self, msg: str) -> str: ...
```

Comment:
Good idea - definitely more in line with modern usage. However; shouldn't the ellipsis go on the next line?

Reply:
Fair enough.